nemo/collections/asr/modules/transformer/transformer_generators.py

```diff
@@ -75,6 +85,7 @@ def _one_step_forward(
         encoder_input_mask=None,
         decoder_mems_list=None,
         pos=0,
+        need_scores: bool = True,
```

Comment:
Can we change the variable to `return_scores` to be consistent with other commands like this?